<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flashcardminigame</class>
 <widget class="QWidget" name="flashcardminigame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="gameProgress">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(216, 216, 216);
font:  700 16pt &quot;Courier New&quot;;</string>
   </property>
   <property name="text">
    <string>GameProgress</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="meaningLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>551</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 15pt &quot;8514oem&quot;;
color: rgb(0, 0, 0);
background-color: rgb(204, 204, 204);

</string>
   </property>
   <property name="text">
    <string>WordLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="feedbackLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>720</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 15pt &quot;Courier New&quot;;</string>
   </property>
   <property name="text">
    <string>FeedBack</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Writethisline">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>570</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);
color: rgb(0, 0, 0);
font: 700 15pt &quot;8514oem&quot;;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>670</y>
     <width>266</width>
     <height>77</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bgflashc/btnnextresized2.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bgflashc/newbgflash.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>gameProgress</zorder>
  <zorder>meaningLabel</zorder>
  <zorder>feedbackLabel</zorder>
  <zorder>Writethisline</zorder>
  <zorder>NextButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
